apps/contracts/src/contracts/SlmChargeback.sol

```diff
@@ -58,17 +58,17 @@ contract SlmChargeback is SlmShared {
     }
 
     /// Allow buyer to withdraw if eligible
-    function buyerWithdraw() external {
-        require(msg.sender == buyer(), 'Only buyer can withdraw');
-        require(judge.voteStatus(address(this)) == 2, 'Cannot withdraw');
+    function buyerWithdraw(bytes32 encryptionKey) external {
+        require(msg.sender == buyer(), "Only buyer can withdraw");
+        require(judge.getVoteResults(address(this), encryptionKey) == 2, "Cannot withdraw");
```

Comment:
Is it possible to reference an enum or imported constant, instead of using integers directly? I can't remember if this would affect anything else, or why I implemented it this way at first.

Same goes for the other similar comparisons.

Reply:
Updated to use enums instead